src/utils/cairoDataTypes/felt252.ts

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable max-classes-per-file */
+
+import { BigNumberish, isBigInt, isBoolean, isHex, isStringWholeNumber } from '../num';
+import { encodeShortString, isLongText, isShortText, isString } from '../shortString';
+
+type ParsableTypes = BigNumberish | Boolean;
```

Comment:
Should this be like this? instead of boolean constructor
```suggestion
type ParsableTypes = BigNumberish | boolean;
```